mock_states/issue_comment.edited.json

```diff
@@ -0,0 +1,27 @@
+
+{
+    "pipeline_statuses": {
+        "default": {
+            "master": "failure"
+        },
+        "buildkite/pipeline2": {
+            "master": "success"
+        }
+    },
+    "issue_tbl" : {
+        "4" : {
+            "comments": {
+                "0": {
+                    "channel": "some channel",
+                    "ts": "some ts"
```

Comment:
Can the mock state be more realistic? The current one should be impossible, since there would never be two different messages posted to the same channel with the same ts. Same for the others